source/usage-examples/count.txt

```diff
@@ -0,0 +1,61 @@
+=============================================
+Count the Number of Documents in a Collection
+=============================================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+The Node.js driver provides two methods for counting the number of
+documents in a collection:
+
+- :node-api:`collection.countDocuments()
```

Comment:
Both of these links just drop you at Collection since ``:node-api:`` points at the 3.0 documentation instead of the most recent doc.

Suggested fix:
- change 3.0 to 3.3 in the definition of ``:node-api:``
- better yet, always point at the latest node-api docs... but I'm not 100% sure if this is possible, or even a good idea

Reply:
I've got a pull in against snooty to update it to 3.3; we can figure out how they do their URLs to see if there's a "latest" or "stable" or something to point to instead.